### PR TITLE
New colours for recovery seed phrase steps

### PR DESCRIPTION
**Pull request by Nebyt**
https://emurgo.atlassian.net/browse/YOEXT-1417

### CODE REVIEW

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js [hunk 1]
```diff
@@ -286,25 +281,22 @@ const InputWrapper = styled(Box)(
...
       !error
         ? `&:not([value=""]):not(:focus) {
         border-color: transparent;
         border: 0;
-        background: ${
-          isVerified
-            ? 'linear-gradient(180deg, #93F5E1 0%, #C6F7ED 100%)'
-            : 'linear-gradient(269.97deg, #E4E8F7 0%, #C6F7ED 100%)'
-        };
+        background: ${isVerified ? theme.palette.ds.el_secondary : theme.palette.primary[100]};
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: maybe we can replace `theme.palette.primary[100]` with `ds` object

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js [hunk 2]
```diff
@@ -286,25 +281,22 @@ const InputWrapper = styled(Box)(
...
+        color: ${isVerified ? theme.palette.static.black : theme.palette.primary[600]}
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: add token from `ds` theme

File: packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js [hunk 3]
```diff
@@ -82,13 +84,20 @@ function VerifyRecoveryPhraseStep(props: Props & Intl): Node {
...
         <Typography component="div" mb="16px">
           <FormattedHTMLMessage {...messages.description} />
         </Typography>
 
-        <Box className={styles.verifyRecoveryPhraseArea}>
+        <Box
+          sx={{
+            border: 'double 2px transparent',
+            borderRadius: '8px',
+            borderColor: 'primary.100',
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js]: can add token from `ds` theme object - `ds.primary_100`

File: packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js [hunk 4]
```diff
@@ -176,17 +183,22 @@ function VerifyRecoveryPhraseStep(props: Props & Intl): Node {
         >
           {sortedRecoveryPhrase.map(({ word, internalWordId }, idx) => {
             const isAdded = addedWordsIndxes.has(idx);
+            const wrongWordBorderColor = (wrondWordId, currentId) => wrongWordIdx === currentId ? 'ds.sys_magenta_500' : 'transparent';
+            const wordBorderColor = (wrondWordId, currentId) => isAdded ? 'primary.200' : wrongWordBorderColor(wrondWordId, currentId);
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js]: add token from `ds` theme object

File: packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js [hunk 5]
```diff
@@ -176,17 +183,22 @@ function VerifyRecoveryPhraseStep(props: Props & Intl): Node {
...
-                className={classnames(styles.wordChip, {
-                  [styles.wordAdded]: isAdded,
-                  [styles.wrongWord]: wrongWordIdx === idx,
-                })}
                 onClick={() => onAddWord(word, idx)}
+                disabled={isAdded}
                 sx={{
+                  height: '40px',
                   width: '123px',
+                  backgroundColor: isAddedOrIsWrong ? 'transparent' : 'primary.100',
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js]: add token from `ds` theme object

File: packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js [hunk 6]
```diff
@@ -181,7 +186,7 @@ export default class RestoreRecoveryPhraseForm extends Component<Props, State> {
                   }}
                   variant="body1"
                 >
-                  <Typography component="div" variant="body1" color="primary.400" width="24px">
+                  <Typography component="div" variant="body1" color="primary.500" width="24px">
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js]: `ds.primary_500`

File: packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js [hunk 7]
```diff
@@ -244,7 +249,13 @@ export default class RestoreRecoveryPhraseForm extends Component<Props, State> {
         <Fade in={isValidPhrase}>
           <Stack gap="10px" direction="row" mt="12px" alignItems="center">
             <VerifiedIcon />
-            <Typography component="div" variant="body1" fontWeight={500} id="validPhraseMessage">
+            <Typography
+              component="div"
+              variant="body1"
+              fontWeight={500}
+              id="validPhraseMessage"
+              color="grayscale.900"
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js]: `ds.gray_900`

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js [hunk 8]
```diff
@@ -197,7 +197,7 @@ function Autocomplete({
         sx={{
           boxShadow: isOpen ? '0px 3px 10px rgba(24, 26, 30, 0.08)' : 'unset',
           maxHeight: 44 * maxVisibleOptions + 'px',
-          color: 'black',
+          color: 'grayscale.900',
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: `ds.gray_900`

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js [hunk 9]
```diff
@@ -206,32 +206,27 @@ function Autocomplete({
...
-              })
+              <Box sx={{ paddingY: '8px' }}>
+                {filteredList.map((item, index) => {
+                  const regularPart = inputValue != null ? item.replace(inputValue.toLowerCase(), '') : item;
+                  return (
+                    <Box
+                      key={`${item}${index}`}
+                      sx={{
+                        padding: '16px',
+                        backgroundColor: highlightedIndex === index ? 'grayscale.50' : 'background.bg_color_max',
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: add token from `ds` theme object

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js [hunk 10]
```diff
@@ -286,25 +281,22 @@ const InputWrapper = styled(Box)(
     padding: 8px;
     letter-spacing: 0;
     text-align: center;
-    border: 2px solid ${error ? '#FF1351' : theme.palette.primary[300]};
+    border: 2px solid ${error ? theme.palette.ds.sys_magenta_500 : theme.palette.primary[300]};
```
Comment (SorinC6) [inline packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: add token from `ds` theme object

File: packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js (hunk 4 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js]: Fixed.

File: packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js (hunk 6 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js]: Fixed.

File: packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js (hunk 7 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/wallet/restore/steps/phrase/RestoreRecoveryPhraseForm.js]: Fixed.

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js (hunk 8 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: Fixed.

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js (hunk 9 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: Fixed.

File: packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js (hunk 10 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/common/autocomplete/Autocomplete.js]: Fixed.

File: packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js (hunk 5 above)
Comment (Nebyt) [reply packages/yoroi-extension/app/components/wallet/create-wallet/VerifyRecoveryPhraseStep.js]: Fixed.

**Comment by Nebyt** [comment]
We need to clarify moment about used color and the titles. Waiting for the designer's response.
[Update]
The response is received. The changes in colours are correct.

**Review by SorinC6 — APPROVED** [review]

**Review by SorinC6 — APPROVED** [review]

**Pull request merged**